grants_tagger/__main__.py

```diff
@@ -233,6 +239,9 @@ def wellcome_science(
     output_path: Optional[Path] = typer.Argument(
         None, help="path to JSONL output file that will be generated"
     ),
+    label_binarizer_output_path: Optional[Path] = typer.Argument(
```

Comment:
you don't like `label_binarizer_path`?

Reply:
haha, yeah, I went with your `output_path` initially, but yep, changed it to be shorter